### ISSUE TITLE
Can 2019 GEDI data be used to tune catastrophic deforestation model?

### ISSUE THREAD

**Issue by crich011** [earthshot-labs/forest#4]
Given a height measurement and other canopy data from 2019 GEDI observations and observed 2018 or 2019 deforestation events from Landsat (verify appropriate time overlap), can we build a model to identify events?

It is possible that this is beyond the scope of MVP work, but I think having the full waveform GEDI data to ground truth our analysis and the Landtrendr outputs (plus any other features we want to include) could allow us to build a really good model. 

Some considerations relating to the data: 

1. We need to ensure that we are using GEDI observations that occur after any candidate events; depending on the time window associated with GEDI data, that might mean using earlier (e.g. 2018) deforestation events as our set of interest.
2. We would need to set thresholds for identifying regions to include and somehow randomize over these regions of interest.

Some considerations relating to the model/analysis:

1. Currently, we use `ltgee.getChangeMap` to identify deforested regions. This takes the output of the Landtrendr algorithm as an input and applies some filtering subject to user-provided hyperparameters. Ideally, we would train a model using the quantities `getChangeMap` cuts on as input features.
2. Unfortunately, I have yet to have any luck finding the source code for `getChangeMap`, but it might be possible to reproduce something like it (or close enough to get inspiration) by looking at the Landtrendr outputs.
3. It's also very possible that someone has already done this, so a healthy amount of Google searching is a good idea. 
4. Model simplicity and interpretability are important considerations here, namely because a simple model that uses only Landtrendr outputs and can provide (potentially regionally-varying) best fit values for `getChangeMap` hyperparameters as an output means we can use `getChangeMap` directly, as opposed to having to apply a CNN or XGBoost model ourselves.

**Comment by crich011** [comment]
This work is currently blocked on the work of #8/#9, but once #9 merges and #2 finishes, we will be able to work on implementing a version of the deforestation model on the full GEDI data.
